src/com/aca/Nare_Silanyan/homework1/models/Manager.java

```diff
@@ -0,0 +1,93 @@
+package com.aca.Nare_Silanyan.homework1.models;
+
+import javafx.util.Pair;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Manager {
+    public static int earned = 0;
+    public static int remained = 0;
+    public List<String> recordItems = new ArrayList<>();
```

Comment:
recordItems property means this is a list of items but its type is List<String>, this is confusing

Reply:
I think this is already done.